strawberry/extensions/tracing/opentelemetry.py

```diff
@@ -76,7 +76,7 @@ async def resolve(self, _next, root, info, *args, **kwargs):
 
             return result
 
-        with self._tracer.use_span(self._root_span):
+        with trace.use_span(self._root_span):
```

Comment:
why was this changed? 😊

Reply:
Looks like it got moved in open-telemetry/opentelemetry-python#1668 :shrug:

Reply:
gotcha! thanks for explaing! 😊